core/src/main/scala/org/apache/spark/storage/ShuffleBlockFetcherIterator.scala

```diff
@@ -457,7 +457,7 @@ final class ShuffleBlockFetcherIterator(
   }
 
   private def createFetchRequests(
-      curBlocks: Seq[FetchBlockInfo],
+      curBlocks: IndexedSeq[FetchBlockInfo],
```

Comment:
Since `createFetchRequests` is a private method, can we directly change the type of `curBlocks` to `Buffer or ArrayBuffer`? This allows Scala 2.12 and 2.13 to achieve optimal performance